cnd/src/config.rs

```diff
@@ -4,31 +4,56 @@ mod settings;
 mod validation;
 
 use crate::ethereum::ChainId;
+use anyhow::{Context, Result};
+use conquer_once::Lazy;
 use libp2p::Multiaddr;
 use reqwest::Url;
 use serde::{Deserialize, Serialize};
-use std::path::PathBuf;
+use std::{fmt::Debug, path::PathBuf, str::FromStr};
 
 pub use self::{
     file::File,
     settings::{AllowedOrigins, Settings},
     validation::validate_connection_to_network,
 };
+use crate::data_dir;
 
-lazy_static::lazy_static! {
-    pub static ref LND_URL: Url = Url::parse("https://localhost:8080").expect("static string to be a valid url");
-}
+static BITCOIND_RPC_MAINNET: Lazy<Url> = Lazy::new(|| parse_unchecked("http://localhost:8332"));
```

Comment:
commit message of e908b28b5cc12bc65e30a6e68e506d7f3689d437 does not match the patch content.

Reply:
Fair, I've probably squashed too much into this one :)